docs/analyst_guide.md

```diff
@@ -235,6 +236,27 @@ Default [Postgres function](https://www.postgresql.org/docs/14/functions-string.
 
 **Restrictions:** In untrusted mode, only `start = 1` is allowed.
 
+## Type casts
+
+When selecting or generalizing columns of personal tables, the following type conversions are allowed.
```

Comment:
Better end this sentence with a colon and list the allowed conversion types as list items.